src/main/kotlin/org/wycliffeassociates/otter/common/domain/usfm/ParseUsfm.kt

```diff
@@ -0,0 +1,89 @@
+package org.wycliffeassociates.otter.common.domain.usfm
+
+import java.io.File
+
+val sep = System.lineSeparator()
+
+val MARKER_BOOK_NAME = "\\id"
+val MARKER_CHAPTER_NUMBER = "\\c"
+val MARKER_VERSE_NUMBER = "\\v"
+var MARKER_NEW_PARAGRAPH = "\\p"
+var MARKER_SECTION_HEADING = "\\s"
+var MARKER_SECTION_HEADING_ONE = "\\s1"
+var MARKER_SECTION_HEADING_TWO = "\\s2"
+var MARKER_SECTION_HEADING_THREE = "\\s3"
+var MARKER_SECTION_HEADING_FOUR = "\\s4"
+var MARKER_CHUNK = "\\s5"
```

Comment:
These should all be vals?

Reply:
Kind of question/suggestion together, but does adding the `private` keyword in front of these constants scope them to just this file (ParseUsfm.kt)?